fe/fe-core/src/main/java/com/starrocks/connector/RemotePathKey.java

```diff
@@ -87,4 +104,18 @@ public String toString() {
         sb.append('}');
         return sb.toString();
     }
+
+    public void drop() {
```

Comment:
clearContext?

Reply:
not only contex.  remote path key is supposed to have more fields attached temporarily.